mongodbatlas/ldap_configurations_test.go

```diff
@@ -133,7 +133,7 @@ func TestLDAPConfigurations_Save(t *testing.T) {
 		}`)
 	})
 
-	request := &LDAP{}
+	request := &LDAPConfiguration{}
```

Comment:
yes, I know. I am not sure how to prevent this issue though 😕 maybe we can populate the struct that we pass to function as well.

Reply:
Yeah, maybe in the test for the call, then at least you purposefully expect certain values to be present in the input. Not blocking this PR, but something to consider.